stats/teams/new_team.php

```diff
@@ -12,9 +12,9 @@
 
 $theme_extra_args = ["js_data" => get_newHelpWin_javascript("$code_url/pophelp.php?category=teams&name=edit_")];
 
-$teamname = stripAllString(trim(array_get($_POST, "teamname", "")));
+$teamname = trim(array_get($_POST, "teamname", ""));
```

Comment:
Does this change alter how team names appear in team forums when they are created? I forget how this crazy team code works but that was the only other phpBB-interfacing piece that came to mind.

Reply:
It seems to display team names OK in the forum too. I created a team `test & test again` on TEST. 
Team details here: https://www.pgdp.org/c/stats/teams/tdetail.php?tid=68
Team forum thread here: https://www.pgdp.org/phpBB3/viewtopic.php?t=752
Searching for `&` on the Team List page finds the new team and displays the name correctly there.

Reply:
I checked the team I created in this sandbox, also, and the team discussion was created correctly (the team that has the `&amp;` explicitly in the team name is one I created in the main test site as a control, not a failure of @windymilla's code).

Reply:
Awesome, thanks for confirming that!